Marsiple/Common/Models/Album.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Album.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+struct Album {
+    let images: [UIImage]
```

Comment:
The `Album` model actually does not contain `UImage` objects, but rather `Photo` objects. Do you plan on changing this later on as you implement the network layer for the `Albums`?

Reply:
Yes I do. This is just for simplifying view testing.